hack/loadtest/loadtest.sh

```diff
@@ -43,27 +44,14 @@ down() {
 
 up() {
   printf "Preparing config\n"
-  mkdir -p "${WORK_DIR}/cerbos"
+  mkdir -p "${WORK_DIR}"/{audit,cerbos}
   mkdir -p "${WORK_DIR}"/postgres/{init,data}
   cp ../../internal/storage/db/postgres/schema.sql "${WORK_DIR}/postgres/init/schema.sql"
-
-  case "$STORE" in 
-    disk)
-      cp conf/cerbos/disk.yml "${WORK_DIR}/cerbos/config.yml"
-      ;;
-
-    postgres)
-      cp conf/cerbos/postgres.yml "${WORK_DIR}/cerbos/config.yml"
-      ;;
-
-    *)
-      echo "Unknown store '$STORE'. Valid values are: 'disk', 'postgres'"
-      usage
-      ;;
-  esac
+  
+  cp conf/cerbos/config.yml "${WORK_DIR}/cerbos/config.yml"
 
   printf "Starting all services\n"
-  docker-compose up -d
+  AUDIT_ENABLED="$AUDIT_ENABLED" SCHEMA_ENFORCEMENT="$SCHEMA_ENFORCEMENT" STORE="$STORE" docker-compose up -d
```

Comment:
We need to pass down `WORK_DIR` here too, otherwise the bind mounts are relative to the filesystem root.

```suggestion
  AUDIT_ENABLED="$AUDIT_ENABLED" SCHEMA_ENFORCEMENT="$SCHEMA_ENFORCEMENT" STORE="$STORE" WORK_DIR="$WORK_DIR" docker-compose up -d
```